lib/email_tracking_retriever.py

```diff
@@ -153,33 +191,34 @@ def get_tracking(self, email_id, mail) -> Tracking:
     group, reconcile = self.get_buying_group(raw_email)
     tracking_number, shipping_status = self.get_tracking_number_from_email(
         raw_email)
+    tracking = Tracking(tracking_number, group, order_ids, price, to_email, '',
+                        date, 0.0, reconcile=reconcile)
     tqdm.write(
-        f"Tracking: {tracking_number}, Order(s): {order_ids}, Group: {group}, Status: {shipping_status}"
+        f"Tracking: {tracking_number}, Order(s): {order_ids}, "
+        f"Group: {group}, Status: {shipping_status}"
     )
-    if tracking_number == None:
-      tqdm.write(
-          f"Could not find tracking number from email with order(s) {order_ids}"
-      )
+    if tracking_number is None:
+      tqdm.write(f"Could not find tracking number from email; we got: {tracking}")
       self.mark_as_unread(email_id)
```

Comment:
should we just do the mark_as_unread step above whenever we've found that get_tracking returned False? It'd mean we don't have to have it in multiple places here

Reply:
Done.